flask_resty/spec/spec_declaration.py

```diff
@@ -0,0 +1,9 @@
+class SpecDeclaration(object):
+
+    def update_spec(self, path, spec, **kwargs):
+        """When overridden, gives a chance to update the path fragment of the
+        specification
+        :param dict path: swagger-spec/blob/master/versions/2.0.md#pathItemObject) # noqa
+        :param apispec.APISpec spec: the specification object
+        """
+        pass
```

Comment:
Still, The inheritance chain could be complicated, and users could not alway know if they are inheriting it or not from some abstract ancestor That doesn't implement it, or something like that. But you have a good point, I guess it's fine to implement it as `raise`